crates/transaction-generator-lib/src/lib.rs

```diff
@@ -124,7 +124,7 @@ impl Default for TransactionType {
             invalid_transaction_ratio: 0,
             sender_use_account_pool: false,
             non_conflicting: false,
-            use_fa_transfer: false,
+            use_fa_transfer: true,
```

Comment:
Doesn't this probably break something else if it's used against a live network?

Reply:
This is used by our benchmark. Who relies on this?